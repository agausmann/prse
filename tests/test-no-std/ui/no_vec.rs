use prse::{parse, Parse};

#[derive(Parse, Eq, PartialEq)]
#[prse = "({:,:})"]
struct Wrapper(Vec<i32>);

fn main() {
    let l = "test: 5,8,9";
    let v: Vec<u32> = parse!(l, "test: {:,:   }");
}
