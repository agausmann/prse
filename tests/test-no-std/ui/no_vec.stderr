error: alloc feature is required to parse into a Vec.
 --> ui/no_vec.rs:4:10
  |
4 | #[prse = "({:,:})"]
  |          ^^^^^^^^^

error: alloc feature is required to parse into a Vec.
 --> ui/no_vec.rs:9:33
  |
9 |     let v: Vec<u32> = parse!(l, "test: {:,:   }");
  |                                 ^^^^^^^^^^^^^^^^
